docs/assets/schemas/api-latest.yaml

```diff
@@ -469,6 +469,10 @@ paths:
             - `17` - Maker lost dispute
             - `18` - Taker lost dispute
 
+            The client can use `cancel_status` to cancel the order only
```

Comment:
This doc is unfortunately auto-generated when running test_trade_pipeline, so these strings here will likely be lost I believe. Maybe I am wrong. I think we should add these helpful comments on `oas_schemas.py` just as #1325 and then the autogenerated `api-latest.yaml` will always have this text.

Reply:
Yes it is auto generated in `tests/test_api.py` at the start of the tests. Also without `ENUM_NAME_OVERRIDES` `api-latest.yaml` gets really polluted :)